src/coordinatesystems.jl

```diff
@@ -17,44 +19,97 @@ function Polar(r, θ)
     return Polar{typeof(r2), typeof(θ2)}(r2, θ2)
 end
 
+# get angle in radians
+angle(x::Polar) = x.θ
+
 Base.show(io::IO, x::Polar) = print(io, "Polar(r=$(x.r), θ=$(x.θ) rad)")
-Base.isapprox(p1::Polar, p2::Polar; kwargs...) = isapprox(p1.r, p2.r; kwargs...) && isapprox(p1.θ, p2.θ; kwargs...)
+Base.isapprox(p1::PolarType, p2::PolarType; kwargs...) = isapprox(p1.r, p2.r; kwargs...) && isapprox(angle(p1), angle(p2); kwargs...)
+
+"""
+`Polard{T,A}(r::T, θ::A)` - 2D polar coordinates (using degrees)
+"""
+struct Polard{T,A} <: PolarType
+    r::T
+    θ::A
+
+    Polard{T, A}(r, θ) where {T, A} = new(r, θ)
+end
+
+function Polard(r, θ)
+    r2, θ2 = promote(r, θ)
+
+    return Polard{typeof(r2), typeof(θ2)}(r2, θ2)
+end
+
+# get angle in radians
+angle(x::Polard) = deg2rad(x.θ)
+
+Base.show(io::IO, x::Polard) = print(io, "Polard(r=$(x.r), θ=$(x.θ)°)")
+
+@inline Base.convert(::Type{Polar}, p::PolarType) = Polar(p.r, angle(p))
+@inline Base.convert(::Type{Polard}, p::Polar) = Polard(p.r, rad2deg(angle(p)))
+
 
 "`PolarFromCartesian()` - transformation from `AbstractVector` of length 2 to `Polar` type"
-struct PolarFromCartesian <: Transformation; end
-"`CartesianFromPolar()` - transformation from `Polar` type to `SVector{2}` type"
+struct PolarFromCartesian{PT<:PolarType} <: Transformation; end
+PolarFromCartesian() = PolarFromCartesian{Polar}() # default is Polar
+"`PolardFromCartesian()` - transformation from `AbstractVector` of length 2 to `Polar` type"
+const PolardFromCartesian = PolarFromCartesian{Polard}
+
+"`CartesianFromPolar()` - transformation from `Polar` or `Polard` type to `SVector{2}` type"
 struct CartesianFromPolar <: Transformation; end
```

Comment:
Did you intend to make this parametric too?

Reply:
I actually spent an hour or so trying to hack this together, but it isn't really necessary and adds a lot of overhead complexity.